docs/design/2018-09-04-histograms-in-plan.md

```diff
@@ -0,0 +1,115 @@
+# Proposal: Maintain statistics in `Plan`
+
+- Author:     Yiding CUI
+- Last updated:  2018-09-04
+
+## Abstract
+
+This proposal proposes to maintain the histogram in `Plan`’s statistics information.
+
+
+## Background
+
+Currently, TiDB only uses statistics when deciding which physical scan method a table should use. And TiDB only stores simple statistics in the plan structure. But when deciding the join order and considering some other optimization rules, we need more detailed statistics.
+So we need to maintain the statistics in the plan structure to get sufficient statistics information to do optimizations.
```

Comment:
You can click "View" in the upper right corner. Then you can see that these two paragraphs are combined together.
Next time, when you want to write another paragraph, please break a line~ Then this article can be displayed normally.